src/raven.js

```diff
@@ -759,7 +760,14 @@ function makeRequest(data) {
     var img = newImage(),
         src = globalServer + authQueryString + '&sentry_data=' + encodeURIComponent(JSON.stringify(data));
 
-    img.crossOrigin = 'anonymous';
+    if ('crossOrigin' in globalOptions) {
```

Comment:
Definitely sounds reasonable to have it undocumented. I added this check because I saw that in the testing, there's assumptions that you can override the `globalOptions` object completely ([there were direct assignments to `globalOptions`](https://github.com/getsentry/raven-js/blob/master/test/raven.test.js#L838-L840)). Doing so would remove the default `globalOptions.crossOrigin`. I was afraid there would be users that relied on this, assuming that all defaults in globalOptions are optional.

But in real-world usage, globalOptions are merged, so that shouldn't be an issue. Hence, I can remove this.